content/posts/2023-04-05-microshift-on-crc.md

```diff
@@ -0,0 +1,34 @@
+---
+title: " Introducing MicroShift preset for CRC"
+date: 2023-04-05T07:12:10+05:30
+---
+
+Are you a developer who wants to use OpenShift Kubernetes but doesn't have a powerful system?
+Or do you need to deploy to a small form factor or edge computing environment? 
+If so, you'll be happy to hear about our new [MicroShift](https://access.redhat.com/documentation/en-us/red_hat_build_of_microshift/4.12/html/getting_started/index) preset,
+a lightweight version of OpenShift Kubernetes that's optimized for resource-limited environments.
+
+The OpenShift preset currently provided by CRC requires 9GB of RAM and 4 CPUs.
+With the release of CRC 2.16, you can now use the MicroShift preset, which only requires 4GB of RAM and 2 CPUs.
+
+What makes MicroShift different from OpenShift Kubernetes? MicroShift is designed for small form factor
```

Comment:
They call it "OpenShift Kubernetes Engine" in the document which is linked  below.